subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -222,3 +229,97 @@ void ull_conn_iso_done(struct node_rx_event_done *done)
 			  ((void *)conn == ull_disable_mark_get()));
 	}
 }
+
+static void ticker_resume_op_cb(uint32_t status, void *param)
+{
+	ARG_UNUSED(param);
+
+	LL_ASSERT(status == TICKER_STATUS_SUCCESS);
+}
+
+static void ticker_resume_cb(uint32_t ticks_at_expire, uint32_t remainder,
+			     uint16_t lazy, uint8_t force, void *param)
+{
+	static memq_link_t link;
+	static struct mayfly mfy = {0, 0, &link, NULL, lll_resume};
+	struct lll_event *resume_event;
+	struct lll_conn_iso_group *cig;
+	uint32_t ret;
+
+	LL_ASSERT(lazy == 0);
+
+	resume_event = param;
+	cig = resume_event->prepare_param.param;
+
+	/* Append timing parameters */
+	resume_event->prepare_param.ticks_at_expire = ticks_at_expire;
+	resume_event->prepare_param.remainder = remainder;
+	resume_event->prepare_param.lazy = 0;
+	resume_event->prepare_param.force = force;
+	resume_event->prepare_param.param = cig;
+	mfy.param = resume_event;
+
+	/* Kick LLL resume */
+	ret = mayfly_enqueue(TICKER_USER_ID_ULL_HIGH, TICKER_USER_ID_LLL,
+			     0, &mfy);
+
+	LL_ASSERT(!ret);
+}
+
+void ull_conn_iso_resume_ticker_start(struct lll_event *resume_event,
+				      uint8_t  resume_cis,
```

Comment:
Would `resume_cis_handle` be a better name for this? Or maybe just `cis_handle`.

Reply:
It is not a handle (this is zero based, handles are offset), so `_idx` or `_index` maybe?

Reply:
Right, I would say `_index`.